src/appshell/commandlinecontroller.h

```diff
@@ -25,12 +25,14 @@
 #include "modularity/ioc.h"
 #include "global/iapplication.h"
 #include "ui/iuiconfiguration.h"
+#include "importexport/iimportexportconfiguration.h"
 
 namespace mu::appshell {
 class CommandLineController
 {
     INJECT(appshell, framework::IApplication, application)
     INJECT(appshell, framework::IUiConfiguration, uiConfiguration)
+    INJECT(appshell, importexport::IImportexportConfiguration, importexportConfiguration)
```

Comment:
Can we make appshellconfiguration and use it instead of uiConfiguration and importexportConfiguration?

Reply:
Interesting idea.
But I think this is an overcomplication, I do not see any benefit, and there is a lack - now we can immediately see in the application of commands what they affect (and this may not only be a change in configurations), we will not see it with appshellconfiguration, we will need to additionally go to appshellconfiguration ...
The CommandLineController itself is intended to be a controller - to determine what needs to be changed